src/Microsoft.Health.Fhir.Liquid.Converter/Filters/CollectionFilters.cs

```diff
@@ -65,23 +65,13 @@ public static object GetIndex(object[] collection, int index)
             return null;
         }
 
-        public static object Slice(object input, int s, int n = 1)
+        public static object SliceArray(object[] input, int s, int n = 1)
         {
             if (input != null)
             {
-                if (input is string inputString)
+                if (input.Count() > s + n)
```

Comment:
The Skip/Take methods below actually work as long as input is not null here. Not saying you need to do this but you could simplify this whole method to one line like this:

```csharp
public static object SliceArray(object[] input, int s, int n = 1)
  => input?.Skip(s).Take(n).ToArray() ?? new object[0];
```

The first null coalescing operator `input?` is the same as checking for null in an if block. The second null coalescing operator `.ToArray() ?? new object[0]` ensures that you're not returning null and instead returning an empty array. Totally up to you if you want to change the implementation, just wanted to share another way of doing the same thing in C#.